azure-pipelines.yml

```diff
@@ -0,0 +1,47 @@
+# NOTE: this Azure configuration is used only for building the sdist and wheels.
+# It uses the OpenAstronomy azure template described in detail at
+# https://openastronomy-azure-pipelines.readthedocs.io/en/latest/
+
+resources:
+  repositories:
+  - repository: OpenAstronomy
+    type: github
+    endpoint: astropy
+    name: OpenAstronomy/azure-pipelines-templates
+    ref: master
+
+# NOTE: for now we only use Azure Pipelines on v* branches and tags
+# for building the source and wheel distributions. If you want to
+# make changes to this configuration via a pull request, you can
+# *temporarily* change the branches include trigger to just '*'
+trigger:
+  branches:
+    include:
+    #- 'v*'
+    - '*'
```

Comment:
I speculate this is here to make sure the tests run it as noted above, but I guess this comment is a reminder to switch it back before merging

Reply:
Yep indeed I'll remove before merging